src/Our.Umbraco.NestedContent/Web/UI/App_Plugins/NestedContent/Js/nestedcontent.controllers.js

```diff
@@ -423,6 +426,18 @@ angular.module("umbraco").controller("Our.Umbraco.NestedContent.Controllers.Nest
             }
             return _p8() + _p8(true) + _p8(true) + _p8();
         };
+
+        var getCurrentNodeIndex = function (node) {
+            return node.id === $scope.realCurrentNode.id;
+        }
+
+        var backupGridConfig = function (prop) {
+            return prop.editor == "Umbraco.Grid" ? prop : false;
```

Comment:
We don't really want a hard dependency on another property-editor, the idea of Nested Content is to be agnostic.

@mattbrailsford - we could look at options to have extensibility points? _events, callbacks?_